ruby_basic/nikolay.braslavets/home_task_1/8.rb

```diff
@@ -0,0 +1,27 @@
+=begin
```

Comment:
Don't use block comments. They cannot be preceded by whitespace and are not as easy to spot as regular comments.
https://github.com/rubocop-hq/ruby-style-guide#no-block-comments

Reply:
Okey, fix this. But I understood, with this version of the comments(begin/end) there might still be problems about the description of the tests with rspec?